lib/api/controllers/documentController.js

```diff
@@ -37,101 +65,101 @@ function DocumentController (kuzzle) {
     }
 
     if (request.input.args.size) {
-      let documentsCount = request.input.args.size - (request.input.args.from || 0);
+      const documentsCount = request.input.args.size - (request.input.args.from || 0);
 
-      if (documentsCount > kuzzle.config.limits.documentsFetchCount) {
-        throw new SizeLimitError(`document:search cannot fetch more documents than the server configured limit (${kuzzle.config.limits.documentsFetchCount})`);
+      if (documentsCount > this.kuzzle.config.limits.documentsFetchCount) {
+        throw new SizeLimitError(`document:search cannot fetch more documents than the server configured limit (${this.kuzzle.config.limits.documentsFetchCount})`);
       }
     }
 
-    return engine.search(request);
-  };
+    return this.engine.search(request);
+  }
 
   /**
    * @param {Request} request
    * @returns {Promise<Object>}
    */
-  this.scroll = function documentScroll (request) {
+  scroll(request) {
     if (!request.input.args.scrollId) {
       throw new BadRequestError('document:scroll must specify a "scrollId" argument');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 86 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)